aws/redshift/schemas.sql

```diff
@@ -0,0 +1,3 @@
+CREATE SCHEMA analysis_pii;
+GRANT ALL PRIVILEGES ON SCHEMA analysis_pii TO GROUP admin;
+GRANT SELECT ON ALL TABLES IN SCHEMA analysis_pii TO GROUP reader_pii;
```

Comment:
If we were to repeat this line and say `GRANT SELECT ON ALL VIEWS`, would that remove the need to do the `GRANT SELECT` at the end of each of the view definitions?

Reply:
Ya, I see what you are saying, let me investigate a little more how we can make this easier.

Docs for reference don't have an immediate answer:
http://docs.aws.amazon.com/redshift/latest/dg/r_GRANT.html